blocks/library/quote/style.scss

```diff
@@ -2,7 +2,7 @@
 	margin: 0 0 16px;
 	box-shadow: inset 0px 0px 0px 0px $light-gray-500;
 
-	footer {
+	cite {
```

Comment:
This change caused a visual regression in the editor, since the citation Editable's `tagName` wasn't updated to `cite`. Possible fix: #4013

Reply:
The CSS selector should have been:
```css
cite,
footer {}
```
... in any case, to not break compat.

Reply:
@jasmussen @karmatosed does this make sense to you?

Reply:
To me it does 👍